.github/actions/tests-ac/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Test Admin Console
+description: Run Test Admin Console
+inputs:
+  sdk-version:
+    description: Javascript SDK version to use
+    required: true
+  cypress-key:
+    description: Cypress record key
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Start Kuzzle stack
+      run: |
+        export GITHUB_DIR=$(pwd)
+        sudo sysctl -w vm.max_map_count=262144
+        npm ci
+      shell: bash
+    - name: Setup Admin console
+      run: |
+        git clone -b 4-dev https://github.com/kuzzleio/kuzzle-admin-console /tmp/admin-console
+        cp -f ./cypress.json /tmp/admin-console/cypress.json
+        cd /tmp/admin-console
+        npm ci
+        npm install kuzzle-sdk-v${{ inputs.sdk-version }}@$GITHUB_DIR
```

Comment:
After investigation last week, we found that there was a problem here with our travis.yml
If npm install failed, the installed sdk still the one wich is in package-lock.json 
So the AC tests will pass but the new sdk version wont be tested

I'm not sure that the problem will happen again using GA so I just let a comment here

Reply:
Yes there was a problem on Travis because we provided relative path to the sdk repository. Now this is an absolute path 😉